packages/legacy/core/App/container-impl.ts

```diff
@@ -115,7 +130,7 @@ export class MainContainer implements Container {
     })
     this._container.registerInstance(TOKENS.NOTIFICATIONS_LIST_ITEM, NotificationListItem)
     this._container.registerInstance(TOKENS.CONFIG, defaultConfig)
-    this._container.registerInstance(TOKENS.COMPONENT_CRED_LIST_HEADER_RIGHT, () => null)
+    this._container.registerInstance(TOKENS.COMPONENT_OPTIONS,defaultStackOptions)
```

Comment:
this._container.registerInstance(TOKENS.COMPONENT_STACK_OPTIONS, defaultStackOptions)